### DISCUSSION TITLE
INITIAL_VALUE is missing on react-svg-pan-zoom

### DISCUSSION THREAD

**Discussion by rubenyeghikyan-highway** [DefinitelyTyped/DefinitelyTyped#58187] [Issues with a @types package]
Hi there. INITIAL_VALUE is missing from [react-svg-pan-zoom](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-svg-pan-zoom)
I got following error when trying to import it
> Module '"react-svg-pan-zoom"' has no exported member 'INITIAL_VALUE'.

I created following PR. Please check and merge if everything is okay
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57944

**Comment by martin-badin** [comment]
@rubenyeghikyan-highway please could you update your PR?

[a comment by an automated account is not shown]
